tests/integration/components/conversations/conversation-part-closed-test.js

```diff
@@ -27,18 +27,42 @@ moduleForComponent('conversations/conversation-part-closed', 'Integration | Comp
   }
 });
 
-test('if current user closes message, "You closed this at" is rendered', function(assert) {
+test('if current user closes message, "You closed this" is rendered', function(assert) {
   assert.expect(1);
 
   let user = {
-    id: 1
+    id: 1,
+    username: 'testuser'
   };
 
+  let closedAt = 'MM/DD/YYYY hh:mm:ss';
+
   set(this, 'author', user);
-  set(this, 'closedAt', moment().subtract(2, 'days'));
+  set(this, 'closedAt', moment().fromNow());
   stubService(this, 'current-user', { user: { id: 1 } });
 
```

Comment:
For the life of me, I can't figure out how to get the timestamp to pass in and have been working on it for an very inefficient length of time, by that I mean all day. This thing has kicked my ass to and fro and I'm sick of looking at it and am going to move to something else so the day isn't a complete wash. Nikola has stepped in a bunch with some guidance, but it didn't seem to help my stupidity.